components/grid/columns/width.md

```diff
@@ -8,25 +8,141 @@ published: True
 
-* When all column widths are explicitly set and the cumulative column width is less than the available Grid width, the remaining width is distributed evenly between all columns.
+## Column Width Behavior
 
-* When only some column widths are set and the cumulative width of the columns with set widths is greater than the available Grid width, a horizontal scrollbar appears and all set column widths are respected. Columns with no set width are invisible as their width is `0`.
+The Grid column width settings can vary and result in the following configurations and behavior:
 
-* When only some column widths are set and the cumulative width of columns with set widths is less than the available Grid width, the widths of the columns with a set width are respected and the remaining width is distributed evenly between the other columns.
+* All Grid columns have set widths:
+    * The sum of the column widths exceeds the Grid component width. The Grid renders a horizontal scrollbar. All column widths match their settings.
+    * The sum of the column widths is less than the Grid width. All columns expand evenly to take up the available space in the Grid.
+* Only some Grid columns have set widths:
+    * The columns that have widths take up less than the Grid component width. The applied column widths match the component settings. The remaining width-less columns shrink or expand, depending on the remaining space. A horizontal scrollbar may appears only if the columns that have widths exceed the Grid width. In this case the remaining columns will disappear.
+    * The columns that have widths take up more than the Grid component width. The applied column widths match the component settings. A horizontal scrollbar appears. All columns without widths shrink to zero width and disappear.
+* No column has a set width: all columns are equally wide. They shrink and expand, depending on the Grid width.
 
-* When no column widths are set, the available width is distributed evenly between all Grid columns.
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
-* To allow the users to auto-fit the column widths to the content, enable [column resizing](slug:components/grid/columns/resize) - a double click on the border between the headers will have the grid adjust the column width according to the size of the data, headers and footers content. It is also possible to [auto-fit columns programmatically](slug:components/grid/columns/resize#autofit-columns).
+<style>
+    .multi-dimensional-table th {
+        padding: 8px;
+        background: var(--bg-table-th-color);
+        color: var(--table-th-color);
+        font-weight: 700;
+        text-align: left;
+        font-size: 18px;
+    }
+</style>
+
+<table class="multi-dimensional-table">
+<tr><th>Explicit&nbsp;Column&nbsp;Widths<br> \ <br> Columns&nbsp;with&nbsp;Widths</th><th>Exceed the Grid Width</th><th>Subceed the Grid Width</th></tr>
```

Comment:
* Consider adjusting the terms. In my eyes, "Explicit Column Widths" and "Columns with Widths" mean the same thing, namely, columns that have set widths.
* I find it hard to understand the table layout. with this first column:
![image](https://github.com/user-attachments/assets/167e9c22-2bef-4616-82cc-ccdb60686ed6)

* "Subceed" is a rarely used term; try going for something more readable.
* To improve the table's readability:
   * Use a parent heading for the two columns on the right.
   * Use a parent heading for the left column.